shared/src/models/appState.ts

```diff
@@ -99,6 +101,8 @@ type AppState = {
   filtersPresets: FiltersPresetsType;
   imagesList: ImagesListType;
   validationIntegration: ValidationIntegration | undefined;
+  feature: Feature | undefined;
+  resourceLink: ResourceLink | undefined;
```

Comment:
why do we need these two properties here?

could we also find another name for `resourceLink` ? something that doesn't contain `resource` since it can easily be confused to the k8s resources..

Reply:
after looking through the entire PR, I don't think we need to store these properties in redux at all, they can be stored in the GettingStarted component and passed as props to children
or use a Context that's available only on that page